website/src/docs/hotchocolate/v10/schema/custom-scalar-types.md

```diff
@@ -73,7 +73,7 @@ This will ensure that the type inference works predictable and will by default i
 
 As I said before in most cases we do not need to do anything since Hot Chocolate has default bindings.
 
-# Any Type
+#  Any Type
```

Comment:
Was this by mistake? 🙂

Reply:
@glen-84 hmm, possibly. Would likely have been caught by a future PR when I enable those rules. 😆 

In the future you may want to use the GitHub suggestion syntax to suggest the change. I'll do that below so that I can click to commit the change.

```suggestion
# Any Type
```